barretenberg/cpp/src/barretenberg/dsl/acir_format/ec_operations.test.cpp

```diff
@@ -20,19 +20,18 @@ class EcOperations : public ::testing::Test {
 
 size_t generate_ec_add_constraint(EcAdd& ec_add_constraint, WitnessVector& witness_values)
 {
-    using cycle_group_ct = bb::stdlib::cycle_group<Builder>;
     witness_values.push_back(0);
```

Comment:
Could you please add tests for all the edgecases? Right now we are only testing 1 case out of all the possibilities

Reply:
I am not sure because the edge cases are not expected to work.